Sparkle/SUFileManager.m

```diff
@@ -11,9 +11,11 @@
 #include <sys/xattr.h>
 #include <sys/errno.h>
 
-#if __MAC_OS_X_VERSION_MAX_ALLOWED < 101000 /* MAC_OS_X_VERSION_10_10 */
-extern NSString *const NSURLQuarantinePropertiesKey WEAK_IMPORT_ATTRIBUTE;
-#endif
+#pragma clang diagnostic push
+// This is a private constant but it's referenced from our unit tests
+#pragma clang diagnostic ignored "-Wmissing-variable-declarations"
+NSString *SUAppleQuarantineIdentifier = @"com.apple.quarantine";
```

Comment:
What do you mean by availability constraints? It's being used on every OS we deploy to. Also, it's not really a URL right? Maybe `SUQuarantinePropertiesKey` or `SUQuarantineExtendedAttributeKey`?